types/artillery/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for artillery 1.6
+// Project: https://github.com/shoreditch-ops/artillery#readme
+// Definitions by: Kira McCoan <https://github.com/kmccoan-allocadia>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as request from 'request';
+import * as events from 'events';
+
+export interface ScenarioContext { vars: {[key: string]: any}; }
+export type Next = (err?: Error) => void;
+export type ResponseRequest = request.ResponseRequest;
+export type RequestResponse = request.RequestResponse;
+export type EventEmitter = events.EventEmitter;
+
+// TypeScript Version: 2.2
```

Comment:
Don't think this is necessary -- but if it were it should go on line 5.

Reply:
🤔  Without it I get a linting error because dtslint uses typescript 2.0 by default & ts.forEachLeadingCommentRange was introduced in typescript 2.1.0

```
TypeError: ts.forEachLeadingCommentRange is not a function
    at /usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:380:16
    at iterate (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:343:20)
    at Array.forEach (<anonymous>)
    at iterate (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:345:50)
    at Array.forEach (<anonymous>)
    at iterate (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:345:50)
    at Array.forEach (<anonymous>)
    at iterate (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:345:50)
    at forEachToken (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:346:7)
    at Object.forEachComment (/usr/local/lib/node_modules/dtslint/typescript-installs/2.0/node_modules/tsutils/util/util.js:378:12)
ts.forEachLeadingCommentRange is not a function
```

If I bump it up to 2.1 then I get a compile error in my test since dotted property for types with string index signatures was only introduce in 2.2.
`ERROR: 5:22  expect TypeScript compile error: Property 'testing' does not exist on type '{ [key: string]: any; }'.`

Moved it to line 5